cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -76,6 +76,40 @@ These roles map to the :ref:`CDAP components <admin-manual-cdap-components>` of
 .. --------------------
 .. include:: ../_includes/installation/hadoop-configuration.txt
 
+#. Set the YARN Application Classpath.
```

Comment:
We should say something more specific like:
Adding additional jars to YARN Application Classpath for Spark jobs.